src/SDKs/SqlManagement/Sql.Tests/Utilities/SqlManagementTestUtilities.cs

```diff
@@ -439,5 +408,24 @@ internal static string GetServerKeyNameFromKeyBundle(KeyBundle keyBundle)
             string vaultName = keyBundle.KeyIdentifier.VaultWithoutScheme.Split('.').First();
             return $"{vaultName}_{keyBundle.KeyIdentifier.Name}_{keyBundle.KeyIdentifier.Version}";
         }
+
+        public static void ExecuteWithRetry(System.Action action, TimeSpan timeout, TimeSpan RecordRetryDelay, Func<CloudException, bool> acceptedErrorFunction)
```

Comment:
1 - Fixed, thanks.
2 - Functionally the same.
3 - The reason I model based on timeout time is that this is being used for operations that are waiting for previously running operations to progress.  The progression of the previous operation is based on time, not number of times I try this operation, which is why I base retry condition off the timeout time.
4 - done.
5 - I don't agree that "return;" is weird in void, as it's the only time it is used, but done.